src/include/platform/NetworkCommissioning.h

```diff
@@ -294,7 +319,15 @@ class ThreadDriver : public Internal::WirelessDriver
         virtual ~ScanCallback() = default;
     };
 
-    virtual Status AddOrUpdateNetwork(ByteSpan operationalDataset) = 0;
+    /**
+     * @brief Adds or updates a Thread network on the device. The driver can fill the outDebugText field to pass any human-readable
+     * messages to the client. On success, the driver should set outNetworkIndex to the index of the network just added or updated.
+     * The value of network index is discarded on failure.
+     *
+     * Note: The outNetworkIndex is 0 when calling this function, so the driver don't need to set the value if they only support one
```

Comment:
Is this note still relevant if all the impls _do_ set it?

Reply:
Removed